db/migrate/20230530101854_create_search_results.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSearchResults < ActiveRecord::Migration[7.0]
+  def change
+    create_table :search_results do |t|
+      t.string :keyword
+      t.integer :search_engine
```

Comment:
Please update the data type if we consider to change the value of enum for `search_engine`